buildpipeline/pipelines.json

```diff
@@ -425,7 +438,42 @@
             "Type": "build/product/",
             "PB_BuildType": "Release"
           }
-        }
+        },
+        {
+          "Name": "Dotnet-CoreClr-Trusted-BuildTests",
+          "Parameters": {
+            "HelixJobType": "test/functional/cli/",
+            "TargetsWindows": "false",
+            "Rid": "rhel6-x64",
+            "TargetQueues": "redhat.69.amd64",
```

Comment:
Are the queue names case-sensitive? If so I think you need to update the casing here.